server/src/main/java/org/eclipse/lsp/cobol/core/visitor/VariableDefinitionDelegate.java

```diff
@@ -128,6 +128,7 @@ public void defineVariable(@NonNull DataDescriptionEntryFormat1Context ctx) {
     checkGlobalFlagFor01Level(variableDefinitionContext);
     setValueClauseText(variableDefinitionContext);
     saveGlobalVariable(variableDefinitionContext);
+    checkRedefinedContainValue(variableDefinitionContext);
```

Comment:
Please, add also the check that the statement is singular as for PIC or VALUE above

Reply:
Done